packages/clay-progress-bar/demo/App.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import ClayProgressBar from '../src/ClayProgressBar';
+
+import 'clay-css/lib/css/atlas.css';
+
+const App: React.FunctionComponent = () => {
+	const [value, setValue] = React.useState(10);
+
+	return (
+		<div>
+			<ClayProgressBar value={30}>{'30%'}</ClayProgressBar>
+
+			<ClayProgressBar status="success" value={50}>
+				{'50%'}
+			</ClayProgressBar>
+
+			<ClayProgressBar feedback status="success" value={100}>
+				{'100% as feedback'}
+			</ClayProgressBar>
+
+			<ClayProgressBar
+				feedback={value === 100}
+				status={value === 100 ? 'success' : 'warning'}
```

Comment:
Why do not these rules stay within the component?

Reply:
We can add them to the component, but I wasn't sure if people would ever use "success" or "warning" purely for the styling and not the semantic.

Reply:
Good question, I have not come across a case yet of using ClayProgressBar.